From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Thu, 29 Jun 2017 22:54:11 +0200
Subject: [PATCH] init: make installation of systemd unit file DESTDIR aware

The installation currently does not care about a DESTDIR, which is
imporant when cross compiling.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 init.d/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.d/Makefile.am b/init.d/Makefile.am
index bf86773176a6..2f92cb7fdde4 100644
--- a/init.d/Makefile.am
+++ b/init.d/Makefile.am
@@ -35,9 +35,9 @@ install-exec-hook:
 
 install-data-hook:
 if ENABLE_SYSTEMD_LOOKUP
-	install -p -m644 haveged.service `pkg-config --variable=systemdsystemunitdir systemd`/haveged.service;
+	install -p -m644 haveged.service $(DESTDIR)`pkg-config --variable=systemdsystemunitdir systemd`/haveged.service;
 else
-	install -p -m644 haveged.service $(unit_dir)/haveged.service;
+	install -p -m644 haveged.service $(DESTDIR)$(unit_dir)/haveged.service;
 endif
 	systemctl enable haveged.service;
 
